website/docs/docs/build/incremental-models.md

```diff
@@ -307,11 +307,12 @@ select ...
 
 ### About incremental_predicates
 
-`incremental_predicates` is an advanced use of incremental models, where data volume is large enough to justify additional investments in performance. This config accepts any valid SQL expression, however dbt does not check the syntax. 
+`incremental_predicates` is an advanced use of incremental models, where data volume is large enough to justify additional investments in performance. This config accepts a list of any valid SQL expression(s). dbt does not check the syntax of the SQL statements. 
 
-For example, this is a pattern we might expect to see on Snowflake:
+For example, this an example of a model configuration in a `yml` file we might expect to see on Snowflake:
```

Comment:
removing 'for example' since it seems superfluous

```suggestion
This an example of a model configuration in a `yml` file we might expect to see on Snowflake:
```